packages/lesswrong/components/form-components/ImageUpload2.tsx

```diff
@@ -0,0 +1,177 @@
+/* global cloudinary */
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { registerComponent } from '../../lib/vulcan-lib';
+import { Helmet } from 'react-helmet';
+import Button from '@material-ui/core/Button';
+import classNames from 'classnames';
+import { cloudinaryCloudNameSetting } from '../../lib/publicSettings';
+import { useTheme } from '../themes/useTheme';
+import { cloudinaryUploadPresetSocialPreviewSetting } from './ImageUpload';
+import { makeCloudinaryImageUrl } from '../common/CloudinaryImage2';
+
+const styles = (theme: ThemeType): JssStyles => ({
+  root: {
+    "& img": {
+      display: "block",
+      marginBottom: 8,
+    },
+  },
+  buttonRow: {
+    margin: 'auto',
+  },
+  button: {
+    background: theme.palette.buttons.imageUpload2.background,
+    "&:hover": {
+      background: theme.palette.buttons.imageUpload2.hoverBackground,
+    },
+    color: theme.palette.text.invertedBackgroundText,
+    textTransform: 'none',
+    margin: 5,
+    fontSize: 14,
+    paddingTop: 10,
+    paddingBottom: 10,
+  },
+  imageBackground: {
+    backgroundSize: 'cover',
+    backgroundPosition: 'top right',
+    width: 370,
+    height: 195,
+    display: 'flex',
+  },
```

Comment:
Is `top right` the right thing to do here? I tested previewing a tall image on Twitter via [this post](https://forum-staging.effectivealtruism.org/posts/ow9NB2tprTfymgXn2/hello-2) and it seems to center the image in the preview? We prob won't match the behavior of all social media platforms, but Twitter might be the most important one to match.

Also I think the size should be responsive, so it doesn't break the layout on mobile